app/build.gradle

```diff
@@ -29,6 +39,26 @@ dependencies {
     implementation 'com.google.android.material:material:1.3.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
     implementation "androidx.viewpager2:viewpager2:1.0.0"
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
+    implementation "androidx.core:core-ktx:1.3.2"
+
+    implementation "com.intuit.sdp:sdp-android:1.0.5"
```

Comment:
What's this SDP lib?

Reply:
It's a very useful library for managing scalable sizes (for either text and images), so you no longer have to harcode dimens in layouts and worry about multi-screen size (at least in most cases).